webpack.config.js

```diff
@@ -34,14 +36,17 @@ module.exports = {
 		} ),
 		new AppCachePlugin(),
 		new HtmlWebpackPlugin( {
-			title: 'Simplenote',
-			templateContent: require( './index-builder.js' ),
-			inject: false
+			title: 'Simplenote'
 		} ),
 		new webpack.DefinePlugin( {
 			'process.env.NODE_ENV': JSON.stringify(
 				process.env.NODE_ENV || 'development'
-			)
+			),
+			config: JSON.stringify( config() )
```

Comment:
If something is released wether a custom build or production build we know the state of the source when it was built.